pkg/skaffold/build/tag/custom.go

```diff
@@ -27,18 +26,18 @@ type CustomTag struct {
 	Tag string
 }
 
-func (c *CustomTag) Labels() map[string]string {
+// Labels are labels specific to the custom tagger.
+func (t *CustomTag) Labels() map[string]string {
 	return map[string]string{
 		constants.Labels.TagPolicy: "custom",
 	}
 }
 
-// GenerateFullyQualifiedImageName tags an image with the custom tag
-func (c *CustomTag) GenerateFullyQualifiedImageName(workingDir, imageName string) (string, error) {
-	tag := c.Tag
+// GenerateTag generates a tag using the custom tag.
+func (t *CustomTag) GenerateTag(workingDir, imageName string) (string, error) {
```

Comment:
nit: using `_` as the name of a parameter is a good way to show that it's unused on purpose, which might be useful here (and everywhere else except the env tagger) since you're not changing the interface to remove the parameter in this PR.

Reply:
The reason I didn't remove the parameter is because envTemplate currently uses imageName and gitCommit uses workingDir. I'll incorporate `_`.